src/pages/DistributePage/DistributePage.tsx

```diff
@@ -183,11 +197,16 @@ const DistributePage = () => {
           </Box>
         </Box>
         <Box css={{ m: '$lg' }}>
-          <AllocationTable users={users} />
+          <AllocationTable
+            users={users as IAllocateUser[]}
+            totalAmountInVault={updateAmount as number}
+            tokenName={selectedVault}
+            totalGive={totalGive as number}
```

Comment:
Because the received gifts can be undefined, we'd have to cast it in this way. I could be wrong though but I think it goes back to the database data type from Hasura generates the types.

Reply:
It would be safest to define `totalGive` such that even in edge cases like that, it defaults to a number value like 0